atmat/lattice/atfittune.m

```diff
@@ -1,14 +1,15 @@
 function newring=atfittune(ring,varargin)
 %ATFITTUNE fits linear tunes by scaling 2 quadrupole families
-% NEWRING = ATFITTUNE(RING,NEWTUNES,QUADFAMILY1,QUADFAMILY2)
+% NEWRING = ATFITTUNE(RING,NEWTUNES,QUADFAMILY1,QUADFAMILY2,TOTALTUNE)
```

Comment:
instead of TOTALTUNE we could use directly NEWTUNES:
* if [0 0]< NEWTUNES <[1 1] then the fast algorithm is used
* if NEWTUNES >= [1 1] then use the phase advance computation